application/view/omeka/migrate/index.phtml

```diff
@@ -1,9 +1,13 @@
-<?php
-$translate = $this->plugin('translate');
-?>
-<h2><?php echo $translate('Update database'); ?></h2>
-<p><?php echo $translate('There was a recent software upgrade and the database needs to be updated. The
-entire site will be down for maintenance until you click the button below.'); ?></p>
+<h2><?php echo $this->translate('Update database'); ?></h2>
```

Comment:
We went through a whole process of adding the `$translate` variables to the views, I don't see much point in taking it out here

Reply:
It's already done. No point putting it back.

Reply:
Let me be clear: I hate our early use of `$foo = $this->plugin('foo');` at the top of the templates. It's unnecessary, hardly saves any space, and is arguably less readable than simply `$this->foo()`. A few years ago I've slowly transitioned away from setting variables, and removed them when appropriate. I believe this time is appropriate as any.